commands/publish/README.md

```diff
@@ -295,6 +296,27 @@ lerna publish --canary --yes
 When run with this flag, `lerna publish` will skip all confirmation prompts.
 Useful in [Continuous integration (CI)](https://en.wikipedia.org/wiki/Continuous_integration) to automatically answer the publish confirmation prompt.
 
+### `--summary-file`
+
+```sh
+lerna publish --canary --yes --summary-file ./output.json
```

Comment:
See comment below on `output.json`, please also make it clear that the value for `--summary-file` is optional and has a default value if not provided

Reply:
Yeah I've simplified this to only accept a dir and have a default output file name